src/sbvr-api/permissions.ts

```diff
@@ -288,9 +289,10 @@ const namespaceRelationships = (
 		if (key === '$') {
 			return;
 		}
+		relationships = relationships as RelationshipInternalNode;
 
 		let mapping = relationship.$;
```

Comment:
This might be better as
```suggestion

		let mapping = (relationship as RelationshipInternalNode).$;
```

Reply:
Oh but it's actually a `RelationshipLeafNode` here..